motion_guide.html

```diff
@@ -3395,8 +3379,7 @@
 
 <p></p>
 The video4linux driver is written by the same author that first created Motion. You can 
-find the source and a brief description at the 
-<a href="http://www.lavrsen.dk/twiki/bin/view/Motion/VideoFourLinuxLoopbackDevice" class="twikiLink">video4linux loopback device web page</a>. 
+find the source within apt packages.  
```

Comment:
I've never used this functionality but these are the packages I was seeing in [jessie](https://packages.debian.org/source/jessie/v4l2loopback) and other releases.